datadog/api/metrics.py

```diff
@@ -106,6 +116,39 @@ def send(cls, metrics=None, **single_metric):
 
         :returns: Dictionary representing the API's JSON response
         """
+        # Set the right endpoint
+        cls._resource_name = cls._METRIC_SUBMIT_ENDPOINT
+        cls._send_common(metrics, **single_metric)
+
+    @classmethod
+    def send_dist(cls, metrics=None, **single_metric):
+        """
+        Submit a distribution metric or a list of distribution metrics to the distribution metric
+        API
+
+        :param metric: the name of the time series
+        :type metric: string
+
+        :param points: a (timestamp, value) pair or list of (timestamp, value) pairs
```

Comment:
@jbarciauskas confirming, this should be `(timestamp, [list of values]) or list of (timestamp, [list of values])` right?

Reply:
Yep